ceph-pr-submodules/build/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd "$WORKSPACE"
+modified_submodules=$(git submodule foreach 'cd $toplevel; git diff --ignore-submodules=dirty origin/master -- $path')
```

Comment:
@andrewschoen the git command itself parses it, it's an option of `git submodule foreach`